lib/scrabble/game.rb

```diff
@@ -0,0 +1,18 @@
+module Scrabble
+  class Game
```

Comment:
The [Mastermind][mastermind] kata described in _The RSpec Book_ defines a `Game` class, which proves being handy to hold the user interaction. The CLI then [delegates to that `Game` class][delegation], which allow to keep it minimal.

  [delegation]: https://github.com/gonzalo-bulnes/kata-mastermind/blob/2a669b67ed9622bc87c41ecfef127add140526f0/mastermind/bin/mastermind#L5-L6
  [game]: https://github.com/gonzalo-bulnes/kata-mastermind/blob/2a669b67ed9622bc87c41ecfef127add140526f0/mastermind/lib/mastermind/game.rb
  [mastermind]: https://github.com/gonzalo-bulnes/kata-mastermind